qa/tasks/cephfs/kernel_mount.py

```diff
@@ -33,11 +33,19 @@ def __init__(self, ctx, test_dir, client_id, client_remote,
         self._mount_bin = ['adjust-ulimits', 'ceph-coverage', self.test_dir +\
                            '/archive/coverage', '/bin/mount', '-t', 'ceph']
 
+    def _enable_debugging(self):
+        shell = """
+sudo modprobe ceph
+echo 'module ceph format "check_new_map mds" +p' | sudo tee /sys/kernel/debug/dynamic_debug/control
+"""
+        self.run_shell_payload(shell, timeout=(5*60))
+
     def mount(self, mntopts=[], check_status=True, **kwargs):
         self.update_attrs(**kwargs)
         self.assert_and_log_minimum_mount_details()
 
         self.setup_netns()
+        self._enable_debugging()
```

Comment:
Should there be `_disable_debugging()` to revert this with `echo '... -p'`, called from `cleanup()`?

Reply:
I don't think so. There could be multiple kclient mounts on a machine. Since we're just statically enabling some debugging, I don't see a point cleaning up.